modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx

```diff
@@ -584,12 +648,49 @@ const ClaimsTable = () => {
 			);
 
 		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [
-		productFilterItems,
-		checkedStateProduct,
-		updateCheckedProduct,
-		filterCheckedLabel,
-	]);
+	}, [productFilterItems, checkedStateProduct, filterCheckedLabel]);
+
+	useEffect(() => {
+		type ActionTypeTest = {eventName: string};
```

Comment:
Please, could you change it?
```suggestion
		type ActionType = {eventName: string};
```

Reply:
Done!